flags.go

```diff
@@ -46,6 +46,7 @@ var decimation = flag.Int("decimation", 1, "integer decimation factor, keep ever
 var timeLimit = flag.Duration("duration", 0, "time to run for, 0 for infinite, ex. 1h5m10s")
 var meterID UintMap
 var meterType UintMap
+var unique = flag.Bool("unique", false, "do not print duplicate values from each meter")
```

Comment:
Should read "do not print duplicate messages from each meter", see other comments.